Tools-Override/tests.targets

```diff
@@ -124,14 +123,27 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="CopyRemoteExecutionConfigFile" 
-          BeforeTargets="GenerateTestExecutionScripts" 
-          Condition="'$(BuildingNETFxVertical)' == 'true' And Exists('$(TestPath)\RemoteExecutorConsoleApp.exe')">
+  <!-- We need to generate a simple config file for desktop test executors to tell them to use DEVPATH environment variable to use their dependencies from there.
+  DEVPATH is being set for desktop runs in RunTests.cmd to the TestHostPath. We need this approach to have desktop Helix runs. -->
+  <Target Name="GenerateDesktopExecutorsConfigFiles"
+          BeforeTargets="GenerateTestExecutionScripts"
+          Condition="'$(BuildingNETFxVertical)' == 'true'">
+    <PropertyGroup>
+      <XunitRunnerNetfxConfigFile>$(TestPath)\$(XunitExecutable).config</XunitRunnerNetfxConfigFile>
+      <RemoteExecutorConfigFile>$(TestPath)\RemoteExecutorConsoleApp.exe.config</RemoteExecutorConfigFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ConfigFileLines Include="&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;" />
+      <ConfigFileLines Include="&lt;configuration&gt;" />
+      <ConfigFileLines Include="&lt;runtime&gt;" />
+      <ConfigFileLines Include="&lt;developmentMode developerInstallation=&quot;true&quot; /&gt;" />
+      <ConfigFileLines Include="&lt;/runtime&gt;" />   
+      <ConfigFileLines Include="&lt;/configuration&gt;" />
```

Comment:
Given this file is constant any reason we just didn't check-in the file and just copy it?